src/main.js

```diff
@@ -0,0 +1,18 @@
+import {NeuralNetworkContext} from './nn/NeuralNetworkContext'
+
+class ML {
+	constructor() {
+    this._nnContext = null;
+  }
+
+  getNeuralNetworkContext() {
+    if (!this._nnContext) {
+      this._nnContext = new NeuralNetworkContext();
+    }
+    return this._nnContext;
+  }
+}
+
+if (typeof navigator.ml === 'undefined') {
```

Comment:
are you using es6? you can do navigator.ml == null

Reply:
Good point. I'll use that.

Reply:
fixed.